examples/docker-build/helm.yaml

```diff
@@ -0,0 +1,6 @@
+image: 
+ name: <Openshift docker registry route>/<your namespace>/myapp
```

Comment:
you should not need to do that, when you push the image to the Openshift image registry, it create an `ImageStream` for it. Here you can simply use:

```
image:
  name: myapp
```

Reply:
I first tried this, but in the generated Deployment we got: image: 'myapp:latest', and not a reference to the actual image. This image is not found. The helm Chart "image" seems to expect a docker image, not an imageStream.

Reply:
mmh, let me check... 

oh you're right, it works when S2I creates the application image because we add the proper annotation to inject the image stream into the deployment https://github.com/wildfly/wildfly-charts/blob/219de25d79b7796a6e5bc187251ec2ec81d4dbb4/charts/wildfly-common/templates/_deployment.yaml#L8-L18

It can still be simplified with using:

```
image:
  name: image-registry.openshift-image-registry.svc:5000/<your namespace>/my-app
```

or directly from the `helm` CLI with 

```
helm install myapp -f helm.yaml --set image.name=image-registry.openshift-image-registry.svc:5000/$OPENSHIFT_NS/my-app wildfly/wildfly
```